### ISSUE TITLE
FluDB

### ISSUE THREAD

**Issue by flaneuse** [biothings/metadataplus#5]
## What is the source's name
FluDB

## What is the web address of the source

https://www.fludb.org/brc/hostFactorExperiments.spg?method=SubmitForm&decorator=influenza&navRoot=true

## Example web address of one dataset

https://www.fludb.org/brc/svExperimentDetails.spg?expAccession=IRD_SV_CA04M001-P&decorator=influenza&context=1579037329929

## Does this source require a user account login? Y/N

- [ ] Yes
- [x] No

## In short describe the importance of this source's addition

Repo provides experiments from NIAID funded studies of viral infection. A given study may be divided into multiple experiments based commonly on the type of biological measurement.

**Comment by flaneuse** [comment]
May be revamped as the BRC websites get updated. #low-priority
